Source/carthage/Outdated.swift

```diff
@@ -7,6 +7,30 @@ import Curry
 
 /// Type that encapsulates the configuration and evaluation of the `outdated` subcommand.
 public struct OutdatedCommand: CommandProtocol {
+	enum UpdateAvailabilityAndApplicability: String {
+		case updatesAvailableAllApplicable =
+		"Will be updated to the newest version."
+		case updatesAvailableSomeApplicable =
+		"Will be updated, but not to the newest version because of the specified version in Cartfile."
+		case updatesAvailableNoneApplicable =
+		"Will not be updated because of the specified version in Cartfile."
+		case noUpdatesAvailable =
```

Comment:
What do you think of renaming this like so?

```swift
public struct OutdatedCommand: CommandProtocol {
    enum UpdateType {
        case newest
        case newer
        case ineligible

        init?(current: PinnedVersion, applicable: PinnedVersion, latest: PinnedVersion) {
            guard current != latest else { return nil }
            if applicable == latest {
                return .newest
            } else if current != applicable {
                return .newer
            } else {
                return .ineligible
            }
        }
    }
}
```

Reply:
Looks good. Your other comments will be covered by this change